shifterator/helper.py

```diff
@@ -72,38 +72,38 @@ def get_score_dictionary(scores):
     Returns
     -------
     type2score: dict
-        Dictionary where keys are types and values are scores of those types
-    lexicon_ref: float
-        If a lexicon was loaded, the middle point of the lexicon's scale.
-        Otherwise, None is returned
+        Keys are types and values are scores of those types
     """
     if isinstance(scores, collections.Mapping):
-        return scores.copy(),None
+        return scores.copy(), None
 
     # Else, load scores from predefined score file in shifterator
     try:
-        lexicon = scores.split('_')[0]
-        score_f = 'lexicons/{}/{}.tsv'.format(lexicon, scores)
-        all_scores = pkgutil.get_data(__name__, score_f).decode('utf-8')
-        if 'labMT' in lexicon:
+        lexicon = scores.split("_")[0]
+        score_f = "lexicons/{}/{}.tsv".format(lexicon, scores)
+        all_scores = pkgutil.get_data(__name__, score_f).decode("utf-8")
+        if "labMT" in lexicon:
             lexicon_ref = 5
-        elif 'SocialSent' in lexicon:
+        elif "SocialSent" in lexicon:
             lexicon_ref = 0
-        elif 'NRC' in lexicon:
+        elif "NRC" in lexicon:
             lexicon_ref = 0.5
     except FileNotFoundError:
-        raise FileNotFoundError('Lexicon does not exit in Shifterator: {}'.format(scores))
+        raise FileNotFoundError(
+            "Lexicon does not exit in Shifterator: {}".format(scores)
```

Comment:
This is minor: if you could fix the typo here ("exists") that would be helpful. I know the typo wasn't introduced by this pull, just noticed as I was going through